doc/user_guide/getting_started.rst

```diff
@@ -295,6 +295,32 @@ to the constructor e.g.
 The numpy array is stored in the :py:attr:`~.signal.Signal.data` attribute
 of the signal class.
 
+.. _example-data-label:
+
+Loading example data and data from online databases
+----------------------------------------------------
+
+HyperSpy is distributed with some example data that can be found in
+`hs.datasets.example_signals`. The following example plots one of the example
+signals:
+
+.. code-block:: python
+
+    >>> hs.datasets.example_signals.EDS_TEM_Spectrum().plot()
+
+.. versionadded:: 0.9
+
+    The :py:func:`~.misc.eels.eelsdb.eelsdb` function in `hs.datasets` can
+    directly load spectra from `The EELS Database <http://eelsdb.eu>`_. For
+    example, the following loads all the boron trioxide spectra currently
+    available in the database:
```

Comment:
Github does not render this paragraph (and the `versionadded`). Is it just github? I guess generally the paragraph should not be indented, as the `versionadded` should only be used for short descriptions, with more info most likely following in a normal text.